### DISCUSSION TITLE
Unable to create any Org in dashboard

### DISCUSSION THREAD

**Discussion by tuppers360** [supabase/supabase#18948] [Questions]
Hi I am trying to create a database and when being asked to create an organisation I get an error no matter what I select whether its free or paid with the following: _'Failed to create organization: Please provide a valid tier.'_

I have never created one before and this is my first account, any idea?

I have created a support ticket but this states - Free plan will receive no guaranteed support response - we try to respond to outage related issues asap however Pro and Enterprise tickets take priority. Upgrade - so I guess this will be days :-( and I would like to start coding today if possible.

**Comment by GaryAustin1** [comment]
I confirmed this is a bug and other users are hitting it.  
I'll update if I see a fix, but you may get response from support first.

**Reply by tuppers360** [reply]
Hi thanks for the update :-) they have got back to me and cant seem to recreate it. It looks like when the account was created when using GitHub it has not created an organisation on the account, not sure how that happened.

**Reply by GaryAustin1** [reply]
Not sure.  It does not work for me also and another user.  I heard from an employee in discord a fix was on the way.  Which would imply they reproduced it.

**Reply by GaryAustin1** [reply]
It just started working for me and was not 5 minutes ago.
Try again.

**Reply by GaryAustin1** [reply]
Oh, also hard refresh the browser.  I did that.

**Reply by tuppers360** [reply]
all working now :-)

**Comment by kevcodez** [comment]
Hi,

apologies for the issues. We've rolled out a fix and should be able to create an organization after reloading the dashboard.
